docs/products/kafka/howto/tiered-storage-overview.rst

```diff
@@ -0,0 +1,73 @@
+Tiered storage overview in Aiven Console
+========================================
+
+Aiven for Apache Kafka® offers a comprehensive overview of tiered storage, allowing you to understand its usage and make informed decisions. This overview provides insights into various aspects of tiered storage, including billing, settings, and storage details.
+
+.. important:: 
+   
+   Aiven for Apache Kafka® tiered storage is an early availability feature, which means it has some restrictions on the functionality and service level agreement. It is intended for non-production environments, but you can test it with production-like workloads to assess the performance. To enable this feature, navigate to the :doc:`Feature preview </docs/platform/howto/feature-preview>` page within your user profile.
+
+
+Access tiered storage overview
+--------------------------------
+
+1. In the `Aiven console <https://console.aiven.io/>`_, choose your project and select your Aiven for Apache Kafka service.
+2. From the left sidebar, select **Tiered Storage**.
+
+   - If you haven't enabled tiered storage for your service, you'll have the option to enable it.
+   - If tiered storage is enabled but not configured for any topics, you have the option to set it up for topics directly. For more details, see :doc:`Enable and configure tiered storage for topics </docs/products/kafka/howto/configure-topic-tiered-storage>`. 
+
+3. Once configured, you can view an overview of tiered storage and its associated details.
+
+
+Key insights of tiered storage
+------------------------------
+
+Get a quick snapshot of the essential metrics and details related to tiered storage:
+
+- **Current billing expenses in USD**: Stay informed about your current tiered storage expenses.
+- **Forecasted month cost in USD**: Estimate your upcoming monthly costs based on current usage.
+- **Data tiered in bytes**: View the volume of data that has been tiered.
```

Comment:
This will need some changes, but let's wait until the Console PR is approved.

Reply:
```suggestion
- **Remote tier usage in bytes**: View the volume of data that has been tiered.
```